pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Adding Kafka information using bindings.
+description: In this tutorial, you'll learn how how to add specific bindings to AsyncAPI document.
+---
+
+## Introduction
+
+In a previous tutorial, you learnt how to manage schemas with schema registry. In this tutorial, you will learn how bindings with Kafka works. You will learn how to define Kafka message key content and extend your AsyncAPI document with protocol-specific information.
+
+## Background Context
+
+Bindings enhances AsyncAPI documentation by providing protocol-specific information. They append to different document sections, such as servers, channels, or messages, and they include standard information that is unique to a particular protocol. 
+
+Bindings improve the understanding and usability of the API by offering more setup options and context for various protocols. It includes information such as topics an application reads from or writes to, how the messages are formatted, and the rules for interacting with various data or messages. 
+
+Bindings configure ecosystems to accommodate specific functionalities not supported by AsyncAPI's core features.
+
+You can configure several objects using Kafka bindings. However, for the scope of this tutorial, we are focusing on two levels of bindings: [server bindings](https://github.com/asyncapi/bindings/tree/master/kafka#server-binding-object) and [channel bindings](https://github.com/asyncapi/bindings/tree/master/kafka#channel-binding-object). 
+
```

Comment:
where to not cover all the bindings levels ? Kafka key bindings are more essential than server bindings for example; if you choose to keep only two levels; do a focus on channel + message key bindings

Reply:
Thanks for this feedback. I will discuss this with Alejandra and prioritize properly.

Reply:
Cool, so Cynthia is just gonna do 3 levels then: server, channel, and messages.